bin/upload_sarif_to_github

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -euo pipefail
+
+sarif_file=$1
+owner=$2
+repo=$3
+
+if [ -z "$sarif_file" ] || [ -z "$owner" ] || [ -z "$repo" ]; then
```

Comment:
If I'm not mistaken, the `set -u` from line 3 will make any of those fail if there is a missing parameter. Because `set -u` will raise an error if the var does not exist, and `$1` won't exist if you don't pass any parameter to the call site.

One way to fix this would be to remove `set -u` but I wouldn't recommend that, because that's a useful safety net to keep.
A better way to fix this would be to use `sarif_file="${1:-}"` instead of `$1` (and same for `$2` and `$3`) so that if `$1` is not defined (missing parameter when calling the script), then `sarif_file` will be set to the empty string—that you can then test using `-z` as you already do here—instead of the script failing during the assignment.

```suggestion
sarif_file="${1:-}"
owner="${2:-}"
repo="${3:-}"

if [ -z "$sarif_file" ] || [ -z "$owner" ] || [ -z "$repo" ]; then
```

Reply:
True 👍 I got from 

```cmd
./upload_sarif_to_github: line 5: $1: unbound variable
```

to 

```cmd
Not enough arguments provided. Usage: ./upload_sarif_to_gh.sh <path to .sarif report> <gh owner> <gh repo>
```

when I run script without expected values. Thanks!